tests/fixtures/response/config/environment.js

```diff
@@ -0,0 +1,49 @@
+module.exports = function(environment) {
```

Comment:
This file should not be required for these tests, since they do not access the `host`